src/modules/blockchainApplication/connection/components/ConnectionProposal/index.js

```diff
@@ -43,13 +66,21 @@ const ConnectionProposal = () => {
           <div>
             <Input
               type="text"
-              onChange={(e) => setValue(e.target.value)}
+              onChange={onInputChange}
               value={value}
               className={styles.input}
               placeholder={t('Enter connection URI')}
             />
-            <span className={styles.feedback}>{status?.message}</span>
-            <PrimaryButton onClick={clickHandler} disabled={value.length === 0 || status.isPending}>
+            {nameSpaceError && (
+              <span className={styles.feedback}>
+                <span className={styles.feedbackErrorColor}>{nameSpaceError}</span>
+                <span> {t('Please enter a supported dApp URI')}</span>
```

Comment:
```suggestion
                <span> {t('Please enter a supported blockchain app URI.')}</span>
```

Since we are naming blockchain app everywhere lets have the same for consistency.